### PR TITLE
Bulk approve/reject actions for Moderators

### PR DESCRIPTION
**Pull request by facuesr**
## What does this PR do?
-Ability to Select All comments and make an action in User History drawer
## How do I test this PR?
-In the User History drawer check 'Select All' and click on approve or reject. 

image

image

Feature #1027

### CODE REVIEW

**Comment by kgardnr** [comment]
This is great @facuesr! Thank you! Once it passes code review, we will merge it 🎉

**Comment by kgardnr** [comment]
Oh and also there is a merge conflict @facuesr, would you mind taking a look?

**Review by kgardnr — APPROVED** [review]

**Comment by facuesr** [comment]
Hi @kgardnr , I've fixed the merge conflict.

**Comment by kgardnr** [comment]
Great, thank you @facuesr!

File: client/coral-admin/src/components/UserDetail.js [hunk 1]
```diff
@@ -55,6 +58,28 @@ export default class UserDetail extends React.Component {
...
+  }
+
+  bulkRejectThenReload = async () => {
+    try {
+      await this.props.bulkReject();
+      this.props.data.refetch();
+    } catch (err) {
+
+      // TODO: handle error.
+      console.error(err);
```
Comment (okbel) [inline client/coral-admin/src/components/UserDetail.js:84]: Could you handle the error with a Notification? 

Look for `import {notify} from 'coral-framework/actions/notification';` in our codebase to find the implementation :)

**Comment by kgardnr** [comment]
Hey @facuesr, @okbel added some feedback for you - let us know if you have any questions!

File: client/coral-admin/src/components/UserDetail.js (hunk 1 above)
Comment (facuesr) [reply client/coral-admin/src/components/UserDetail.js:84]: Sure, I will add the notification.  I based the bulkAcceptThenReload and  bulkRejectThenReload functions  in rejectThenReload and acceptThenReload functions. Those function does not have the notifiation neither, do you want that I add a notification there too?

File: client/coral-admin/src/components/UserDetail.css [hunk 2]
```diff
@@ -116,29 +116,50 @@
...
     width: 48px;
     height: 48px;
     transform: scale(.7);
     min-width: 0;
   }
-
   .bulkAction:last-child {
     margin-left: -10px;
   }
+  span {
```
Comment (alinhle) [inline client/coral-admin/src/components/UserDetail.css]: Can you make a classname out of this with a nice descriptive name?

File: client/coral-admin/src/components/UserDetail.js (hunk 1 above)
Comment (alinhle) [reply client/coral-admin/src/components/UserDetail.js:84]: That would be terrific! `notify` is a redux action that needs to be mounted. Then just call `this.props.notify('error', getErrorMessages(err));`. `getErrorMessages` is in `import {getErrorMessages} from 'coral-framework/utils'`

File: client/coral-admin/src/components/UserDetail.css (hunk 2 above)
Comment (facuesr) [reply client/coral-admin/src/components/UserDetail.css]: sure!

**Comment by facuesr** [comment]
hi @kgardnr , I've added the changes.

**Comment by kgardnr** [comment]
Great @facuesr! We will take a look -- thanks!

**Review by alinhle — APPROVED** [review]
Thanks @facuesr!

**Pull request merged**